tests/benchmarks/footprints/src/timer.c

```diff
@@ -97,7 +97,7 @@ void run_timer(void)
 
 	k_mem_domain_add_thread(&footprint_mem_domain, tid);
 
-	k_thread_access_grant(tid, &timer0);
+	k_thread_access_grant(tid, (&timer0));
```

Comment:
I feel like this is double. The macro `k_thread_access_grant` itself is changed to have extra parenthesis, so why does this have to change as well?

Reply:
k_thread_access_grant has been changed to add parenthesis for its first argument, its definition cannot be changed to add parenthesis for its variadic arguments.

Reply:
Ok, that makes sense.